app/main/views/feedback.py

```diff
@@ -553,6 +546,16 @@ def needs_escalation(ticket_type, severe):
     )
 
 
+def is_emergency_ticket(ticket_type, severe):
+    if ticket_type != PROBLEM_TICKET_TYPE or not severe:
+        return False
+
+    if current_user.is_authenticated:
+        return True
+
+    return not in_business_hours()
```

Comment:
```suggestion
    if current_user.is_authenticated or not in_business_hours():
        return True

    return False
```

Do you think we should change this slightly to make it more readable?